### PR TITLE
Polymer with jwt api

### PR DESCRIPTION
**Pull request by kmaida**
Polymer article: **Build a Polymer App: From Authentication to Calling an API**

### CODE REVIEW

File: _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown [hunk 1]
```diff
@@ -0,0 +1,1363 @@
+---
+layout: post
+title: "Build a Polymer App: From Authentication to Calling an API"
+description: "Build an app using web components and Polymer with an API and JSON Web Token authentication."
```
Comment (kukicado) [inline _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown]: This reads a little weird.. maybe
Build an app using web components with polymer that interacts with an API and has JSON Web Token authentication. 
WDYT?

File: _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown [hunk 2]
```diff
@@ -0,0 +1,1363 @@
...
+  name: "Kim Maida"
+  url: "https://twitter.com/KimMaida"
+  mail: "kim.maida@auth0.com"
+  avatar: "https://en.gravatar.com/userimage/20807150/4c9e5bd34750ec1dcedd71cb40b4a9ba.png"
+design:
+  image: https://cdn.auth0.com/blog/polymer/Polymer-Logo-blog.png
+  bg_color: "#324090"
+tags:
+- polymer
+- web components
```
Comment (kukicado) [inline _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown]: web componets => web-components (can't have spaces here)

File: _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown [hunk 3]
```diff
@@ -0,0 +1,1363 @@
...
+
+The [Polymer Element Catalog](https://elements.polymer-project.org/) provides many prepackaged [custom elements that can be easily implemented](https://elements.polymer-project.org/guides/using-elements) in any Polymer project. They provide useful utility elements like [Ajax](https://elements.polymer-project.org/elements/iron-ajax), [media queries](https://elements.polymer-project.org/elements/iron-media-query), and [single page application routing](https://elements.polymer-project.org/elements/app-route) as well as the [visual "Paper" elements](https://elements.polymer-project.org/browse?package=paper-elements) of [Google's Material Design](https://material.google.com/).
+
+Polymer has comprehensive documentation at the [Polymer Project Devguide](https://www.polymer-project.org/1.0/docs/devguide/feature-overview) and can be used with or without the custom elements in the Element Catalog. To speed development, we'll build our simple Single Page Application (SPA) with a starter kit and Polymer elements.
```
Comment (kukicado) [inline _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown]: To speed => To speed up

File: _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown [hunk 4]
```diff
@@ -0,0 +1,1363 @@
...
+  |-- register-login.html
+  |-- secret-quotes.html
+  |-- not-found.html
+```
+
+Our final file `/src` folder file structure should look like this:
+
+![Polymer starter kit Chuck Norris app src file structure](https://cdn.auth0.com/blog/polymer/rename-files2.jpg)
+
+**Editing the Views**
```
Comment (kukicado) [inline _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown]: Should this be a heading?

File: _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown (hunk 1 above)
Comment (kmaida) [inline _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown]: 👍 

How about:

```
Build an app using web components and Polymer that interacts with an API and authenticates users with JSON Web Tokens.
```

File: _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown (hunk 2 above)
Comment (kmaida) [inline _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown]: 👍

File: _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown (hunk 4 above)
Comment (kmaida) [inline _posts/2016-10-05-build-a-polymer-app-from-authentication-to-calling-an-api.markdown]: When it was a heading, it displayed extremely oddly on the rendered blog page. I think the blog engine doesn't handle `h4`s very well. I think I may just remove these headings altogether.

**Pull request merged**